Kinetic_shape_partition/include/CGAL/KSP_3/Finalizer.h

```diff
@@ -116,11 +116,6 @@ class Finalizer {
 
     create_volumes();
```

Comment:
The files which are not part of the public API should go in a directory and namespace `KSP_3/internal`